praw/models/reddit/message.py

```diff
@@ -45,6 +45,10 @@ def __init__(self, reddit, _data):
         super(Message, self).__init__(reddit, _data)
         self._fetched = True
 
+    def delete(self):
+        """Delete the message from the inbox"""
```

Comment:
Great work! 2 very minor things:
- Can you add a period at the end of the dosctring?
- Can you remove the space between the bracket and the parenthesis at the end of line 50?

Thanks